airflow/contrib/operators/redis_publish_operator.py

```diff
@@ -61,6 +61,6 @@ def execute(self, context):
 
         self.log.info('Sending messsage %s to Redis on channel %s', self.message, self.channel)
 
-        result = redis_hook.get_conn().publish(channel=self.channel, message=self.message)
+        result = redis_hook.publish_message(channel=self.channel, message=self.message)
```

Comment:
The result will always be `None` because the hook method does not return anything

Reply:
i've changed publish_message to return int (as mentioned above) so now it makes sense to capture result